apps/stage-tamagotchi/scripts/build-by-platforms.ts

```diff
@@ -0,0 +1,43 @@
+import process from 'node:process'
+
+import { execSync } from 'node:child_process'
+import { join } from 'node:path'
+
+try {
+  if (process.platform === 'linux') {
+    // Tauri sets these environment variables during the build process.
+    const appDir = process.env.TAURI_BUNDLER_APPDIR
+    const linuxdeployPath = process.env.TAURI_BUNDLER_LINUXDEPLOY_BIN
+
+    // Verify that the necessary paths are available.
+    if (!appDir || !linuxdeployPath) {
+      console.error('Error: Required Tauri build environment variables are missing.')
+      console.error(`TAURI_BUNDLER_APPDIR: ${appDir}`)
+      console.error(`TAURI_BUNDLER_LINUXDEPLOY_BIN: ${linuxdeployPath}`)
+      process.exit(1)
+    }
+
+    console.log(`[AIRI-AppImage] Fixing AppImage at: ${appDir}`)
+
+    // Set environment variables for the GStreamer plugin.
+    const env = { ...process.env }
+    env.LINUXDEPLOY = linuxdeployPath
+    const arch = process.arch === 'x64' ? 'x86_64' : process.arch
```

Comment:
```suggestion
    const arch = process.arch === 'x64' ? 'x86_64' : process.arch
```

What about `arm64` or `aarch64`.

Reply:
Now it's lunchtime so I cannot directly patch. But It should be aarch64.

arm64->aarch64 will be okay

Reply:
I mean, I don't mean to patch it with rename, I mean, will this cause `arm64` or `aarch64` Linux fail to build?

Reply:
In fedora asahi remix 42, `strip` command failed with RELR error.
However, RPM Redhat package builder is working well.

Reply:
How about making new issue for aarch64 strip error?
It may vary by each distros